### PR TITLE
Время разбивать камни

### PR DESCRIPTION
**Pull request by caesarmind**
---
:mortar_board: [Время разбивать камни](https://up.htmlacademy.ru/ecmascript/16/user/846107/tasks/3)

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import {listSorting, sortingEvents} from './view/list-sorting.js';
```
Comment (yakovina) [inline src/main.js:1]: you have linter error
please, check

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+};
+
+
+const tripMenu = document.querySelector('.trip-controls__navigation');
+const tripMain = document.querySelector('.trip-main');
+const tripControlsFilter = document.querySelector('.trip-controls__filters');
+
+const tripEvents = document.querySelector('.trip-events');
+const tripPointsList = document.querySelector('.trip-events__list');
+
```
Comment (yakovina) [inline src/main.js:25]: It will be more readable and scalable to add tripList as component, just suggestion

File: src/main.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import {listSorting, sortingEvents} from './view/list-sorting.js';
+import { filteringList } from './view/list-filter.js';
+import { pointList } from './view/list-content.js';
+import { tripNavigation } from './view/menu.js';
+import { editTrip } from './view/edit-trip.js';
+import { tripInfo } from './view/trip-info.js';
```
Comment (yakovina) [inline src/main.js:6]: Please, rename functions as verbs (that answer the question "what to do". )

classes should be nouns, functions - verbs

File: public/index.html [hunk 4]
```diff
@@ -34,7 +34,16 @@ <h2 class="visually-hidden">Filter events</h2>
     <main class="page-body__page-main  page-main">
       <div class="page-body__container">
           <section class="trip-events">
+            <h2 class="visually-hidden">Trip events</h2>
+            <ul class="trip-events__list">
+
+
+
+            </ul>
```
Comment (yakovina) [inline public/index.html:42]: accualy, you can delete comments for now)

File: src/main.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+
+renderTemplate(tripMain, tripInfo(), renderPos.AFTERBEGIN);
+renderTemplate(tripMenu, tripNavigation(), renderPos.BEFOREEND);
+renderTemplate(tripControlsFilter, filteringList(), renderPos.BEFOREEND);
+
+renderTemplate(tripEvents, listSorting(), renderPos.AFTERBEGIN);
+renderTemplate(tripPointsList, editTrip(), renderPos.AFTERBEGIN);
+for(let i = 0; i < 3; i++){
+  renderTemplate(tripPointsList, pointList(), renderPos.BEFOREEND);
```
Comment (yakovina) [inline src/main.js:34]: please, add constants for "magic numbers"

File: src/main.js (hunk 2 above)
Comment (caesarmind) [reply src/main.js:25]: It's component already.

File: src/main.js (hunk 3 above)
Comment (caesarmind) [reply src/main.js:6]: > 

Should name them as verbs even if they are components?

File: src/main.js (hunk 2 above)
Comment (caesarmind) [reply src/main.js:25]: Named as a "list-filter" - or do u meant another thing?

**Pull request merged**